kimchi/js/bindings.js

```diff
@@ -1219,6 +1235,46 @@ var caml_fq_plonk_gate_to_rust = function (gate) {
   );
 };
 
+// Provides: caml_fp_lookup_table_to_rust
+// Requires: plonk_wasm, caml_fp_vector_of_rust
+var caml_fp_lookup_table_to_rust = function (caml_lookup_table, mk_class) {
```

Comment:
Done in 96ba0aeae59381a32579ba144447ea0145214e70. However, it was there because this PR is built on top of others which introduces it. Will add it back later.